### DISCUSSION TITLE
Question about T41U5XBB

### DISCUSSION THREAD

**Discussion by tonluc123** [phil-barrett/grblHAL-teensy-4.x#65] [Q&A]
Hello,
I just obtained the board and I was wondering if the scenario of powering with 12Vdc for spindle control excludes powering Teensy with external 5Vdc at the same time (since jumper is for 1 of those 2 cases). Is it possible - recommended  to do this by "cutting" power from the usb cable and solder the external power at teensy's 5Vin pin ?
Lucas

**Comment by phil-barrett** [comment]
No, it does not exclude use of 5V.  The 5V and 12V circuits can be powered separately.  The 5V/12V jumper only affects the relay power circuits.

Be aware that powering the teensy via 5V and using USB can create a conflict (USB supplies power).  Your options:

- Cut the +5V trace on the T41U5XBB (see manual for "External +5V Feed"
- Cut the Teensy 4.1 trace (see PJRC.com documentation)
- Use a data only USB cable.

**Comment by tonluc123** [comment]
Thanks a lot Phil
